<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="india-its-mbp-6.global.tektronix.net" ignored="0" name="Test" tests="16" failures="7" timestamp="08 Aug 2018 13:44:51 GMT" time="978.996" errors="0">
  <testcase name="f" time="0.005" classname="TestNGExample.ParallelTest"/>
  <testcase name="g" time="0.01" classname="TestNGExample.AlltestNGMethod"/>
  <testcase name="f" time="0.013" classname="TestNGExample.AlltestNGMethod">
    <failure type="java.lang.AssertionError" message="expected [Expected Value] but found [Actaul value]">
      <![CDATA[java.lang.AssertionError: expected [Expected Value] but found [Actaul value]
at TestNGExample.AlltestNGMethod.f(AlltestNGMethod.java:18)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.001" classname="TestNGExample.ParallelTest"/>
  <testcase name="f" time="0.001" classname="TestNGExample.ParallelTest"/>
  <testcase name="testValidatePrime" time="0.004" classname="DataProvider.PNCDataPrivider"/>
  <testcase name="f" time="0.002" classname="TestNGExample.AlltestNGMethod">
    <failure type="java.lang.AssertionError" message="expected [Expected Value] but found [Actaul value]">
      <![CDATA[java.lang.AssertionError: expected [Expected Value] but found [Actaul value]
at TestNGExample.AlltestNGMethod.f(AlltestNGMethod.java:18)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="testValidatePrime" time="0.002" classname="DataProvider.PNCDataPrivider"/>
  <testcase name="f" time="0.001" classname="TestNGExample.AlltestNGMethod">
    <failure type="java.lang.AssertionError" message="expected [Expected Value] but found [Actaul value]">
      <![CDATA[java.lang.AssertionError: expected [Expected Value] but found [Actaul value]
at TestNGExample.AlltestNGMethod.f(AlltestNGMethod.java:18)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="testValidatePrime" time="0.002" classname="DataProvider.PNCDataPrivider"/>
  <testcase name="f" time="0.002" classname="TestNGExample.AlltestNGMethod">
    <failure type="java.lang.AssertionError" message="expected [Expected Value] but found [Actaul value]">
      <![CDATA[java.lang.AssertionError: expected [Expected Value] but found [Actaul value]
at TestNGExample.AlltestNGMethod.f(AlltestNGMethod.java:18)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="testValidatePrime" time="0.002" classname="DataProvider.PNCDataPrivider"/>
  <testcase name="testValidatePrime" time="0.001" classname="DataProvider.PNCDataPrivider"/>
  <testcase name="testAlert" time="12.064" classname="TestNGExample.TestAlert">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestNGExample.TestAlert.testAlert(TestAlert.java:21)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAlert -->
  <testcase name="lounchWebsite" time="978.948" classname="Example.DummyClass">
    <failure type="org.openqa.selenium.WebDriverException" message="Reached error page: about:neterror?e=dnsNotFound&amp;amp;u=http%3A//automationpractice.com/index.php&amp;amp;c=UTF-8&amp;amp;f=regular&amp;amp;d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20automationpractice.com.
Build info: version: &amp;apos;3.13.0&amp;apos;, revision: &amp;apos;2f0d292&amp;apos;, time: &amp;apos;2018-06-25T15:32:14.902Z&amp;apos;
System info: host: &amp;apos;india-its-MacBook-Pro-6.local&amp;apos;, ip: &amp;apos;Unknown&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.13.2&amp;apos;, java.version: &amp;apos;1.8.0_172&amp;apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 61.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 3581, moz:profile: /var/folders/bx/vfbpwh2n755..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: MAC, platformName: MAC, platformVersion: 17.3.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 5c98a400-7beb-bc41-b267-ce02b37322a8">
      <![CDATA[org.openqa.selenium.WebDriverException: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//automationpractice.com/index.php&c=UTF-8&f=regular&d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20automationpractice.com.
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'india-its-MacBook-Pro-6.local', ip: 'Unknown', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.2', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 61.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 3581, moz:profile: /var/folders/bx/vfbpwh2n755..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: MAC, platformName: MAC, platformVersion: 17.3.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 5c98a400-7beb-bc41-b267-ce02b37322a8
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:276)
at com.sellabs.actionable.MainClass.loadWebSite(MainClass.java:50)
at Example.DummyClass.lounchWebsite(DummyClass.java:23)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- lounchWebsite -->
  <testcase name="testDoLogin" time="978.948" classname="com.sellabs.testexecutor.TestMainClass">
    <failure type="org.openqa.selenium.WebDriverException" message="Reached error page: about:neterror?e=dnsNotFound&amp;amp;u=http%3A//automationpractice.com/index.php&amp;amp;c=UTF-8&amp;amp;f=regular&amp;amp;d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20automationpractice.com.
Build info: version: &amp;apos;3.13.0&amp;apos;, revision: &amp;apos;2f0d292&amp;apos;, time: &amp;apos;2018-06-25T15:32:14.902Z&amp;apos;
System info: host: &amp;apos;india-its-MacBook-Pro-6.local&amp;apos;, ip: &amp;apos;Unknown&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.13.2&amp;apos;, java.version: &amp;apos;1.8.0_172&amp;apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 61.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 3583, moz:profile: /var/folders/bx/vfbpwh2n755..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: MAC, platformName: MAC, platformVersion: 17.3.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 78f67b4b-6e83-5f43-b921-af674704c225">
      <![CDATA[org.openqa.selenium.WebDriverException: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//automationpractice.com/index.php&c=UTF-8&f=regular&d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20automationpractice.com.
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'india-its-MacBook-Pro-6.local', ip: 'Unknown', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.2', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 61.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 3583, moz:profile: /var/folders/bx/vfbpwh2n755..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: MAC, platformName: MAC, platformVersion: 17.3.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 78f67b4b-6e83-5f43-b921-af674704c225
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:276)
at com.sellabs.actionable.MainClass.loadWebSite(MainClass.java:50)
at com.sellabs.testexecutor.TestMainClass.testDoLogin(TestMainClass.java:22)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- testDoLogin -->
</testsuite> <!-- Test -->
